modules/javafx.controls/src/test/java/test/javafx/scene/control/AccordionTest.java

```diff
@@ -55,7 +54,7 @@ public class AccordionTest {
     private StackPane root;
 
     @Before public void setup() {
-        tk = (StubToolkit)Toolkit.getToolkit();//This step is not needed (Just to make sure StubToolkit is loaded into VM)
+        tk = Toolkit.getToolkit();//This step is not needed (Just to make sure StubToolkit is loaded into VM)
```

Comment:
not equivalent.
I wonder if we need an assertTrue here

Reply:
Not needed in the setup, if it is `null` later or throws an exception the test failure is clear enough.

Reply:
Yes, adding an assert does seem out of scope of this PR, although Andy comment points out that this is not guaranteed to be behavior neutral. If for some reason the returned type cannot be cast to StubToolkit, the former code would throw CCE while the modified code will not. The purpose of that call might have been merely to ensure that the toolkit was loaded, but it's also possible that it was intended to ensure that the toolkit was an instance of StubToolkit (which it will be absent a bug in how the tests are launched...many things will break if it isn't).

Reply:
This looks like bad test design. If the toolkit needs to be of a specific type when running tests, then check that this is the case once before running all the tests. As for this PR, I don't mind leaving it as is or reverting these changes. I don't think that a CCE is realistic here.

Reply:
> As for this PR, I don't mind leaving it as is or reverting these changes. I don't think that a CCE is realistic here.

I agree that this isn't a good test design. I was just pointing out why it isn't (in theory) a completely compatible change. I don't really mind one way or the other here.

Reply:
I can replace these with an assert:

    assertTrue(Toolkit.getToolkit() instanceof StubToolkit);

Reply:
I wouldn't recommend that, since it will then obscure the main reason for this call -- to load the toolkit (honestly, the cast was probably an afterthought). So this would be OK:

```
        tk = Toolkit.getToolkit();//This step is not needed (Just to make sure StubToolkit is loaded into VM)
        assertTrue(tk instanceofStubToolkit);
```

Reply:
could we just create a method in Util instead of dragging multi-line construct across the tests?

e.g. `Util.loadStubToolkit()` ?

Reply:
Yes, but this is now getting to be a bit out of scope for this PR. Even adding the `assertTrue` requires running all the tests (which thankfully, GHA will do).

Perhaps a better approach is to not make any more changes here, and file a follow-on bug to A) add a utility method, and B) call it from a `@BeforeClass` method (no need to call it more than once per class).

Reply:
> I wouldn't recommend that, since it will then obscure the main reason for this call -- to load the toolkit (honestly, the cast was probably an afterthought). So this would be OK:
> 
> ```
>         tk = Toolkit.getToolkit();//This step is not needed (Just to make sure StubToolkit is loaded into VM)
>         assertTrue(tk instanceof StubToolkit);
> ```

Sorry, I should be more clear, I meant to replace it with what you wrote (including the comment, although perhaps rephrase that to "Ensure StubToolkit is loading into the VM", as the "not needed" bit is clearly false.

As for further optimization with BeforeClass, I think we may be overthinking this -- it is just a getter (after the first call), and moving it to BeforeClass will require even more changes (the field would need to become static).

Reply:
I've adjusted all of these now to either a check that it is a StubToolkit, or if the variable was actually used, a check if the variable is of the correct type.